macos-core-installation.md: Fix paths for brew packages (#874)
azerothcore/wiki: docs/server-setup.md

1. Download the files above.

> 1. Create a new folder within the root of your repo called **Data**. i.e **E:\azerothcore-wotlk\data\ (Windows)** or **/home/$USER/azerothcore/data (Linux/macOS)** (if you want to keep multiple sets of data you may want to create further levels below this, but if this is not explicitly what you need just leave it all flat under your newly created data folder).

Reviewer: this is just wrong

Reviewer: ```suggestion
1. Create a new folder within the build directory called **Data**. i.e **$\Build\RelWithDebInfo\Data**
```

Reviewer: Why I say it is wrong is because the config option default points to "." which is the same directory where the worldserver is. So \build\RelWithDebInfo\

1. Extract the files from the zip file and place them within the **Data** folder.

> 1. Edit your the [DataDir](#updating-datadir) config option to point to the location of the newly created folder from step 2.

Reviewer: Don't make it wordy, there's no reason for it. Keep it short and concise.

By default you will compile your core with tools and you will get the following executable files: **mapextractor**, **vmap4extractor**, **vmap4assembler**, **mmaps_generator** (.exe on windows).

> Place the files in the same directory as your World of Warcraft binary (wow.exe on windows) and run them one after the other.

Reviewer: You can run them at the same time. Except vmap4assambler which needs to be run after the extractor.

> After extracting all necessary files, create a folder called **Data** (as created in step 2 of the previous section) and place files in there. Alternatively you can specify a different directory where you want to keep them by changing DataDir value in the [DataDir](#updating-datadir) worldserver.conf file.

Reviewer: You can't point to a previous step that isn't within the scope of the topic.
`##Download` & `##Extractors (Not needed if you downloaded the files above)` are two different topics.

Also keep the RelWithDebInfo and Debug stuff.

1. In your worldserver.conf file locate **DataDir** option.

> 1. Edit it to the path of your Data folder (azerothcore/Data by default if you followed the steps above)

Reviewer: ```suggestion
1. Edit it to the path of your Data folder. i.e **$\Build\RelWithDebInfo\Data**```
```
Keep it simple and concise. Good to keep the example, but by replacing E:\ to $ we can include all OS'

Reviewer: see https://www.azerothcore.org/wiki/sql-directory
